src/test/java/org/candlepin/resource/EnvironmentResourceTest.java

```diff
@@ -241,10 +240,6 @@ void shouldCreateConsumerInMultipleEnvironment() {
         assertEquals(dto.getEnvironments().get(0).getId(), "env1");
         assertEquals(dto.getEnvironments().get(1).getId(), "env3");
         assertEquals(dto.getEnvironments().get(2).getId(), "env2");
-
-        // TODO this need to be changes whenever we make changes to
-        // consumerDTO to handle 'environment'
-        assertNull(dto.getEnvironment());
```

Comment:
I think instead of deleting this we could simply verify that the `dto.getEnvironment().getName()` is populated as expected?

Reply:
Since this `environment` gets populated during DTO translation, this might not be right place to test this. I have moved this testing bit in [ConsumerTranslatorTest.java](https://github.com/candlepin/candlepin/pull/3162/files#diff-bea22202f35772a874ecbaa6bc0ecb1d84e9401e2e267eff80ead6c32cb5f522R258)